routes/productRouter.mjs

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { ProductController } from '../controllers';
+
+const productRouter = express.Router();
+const productController = new ProductController();
+
+productRouter.get('/products', (req, res) => productController.getAllProducts(req, res));
```

Comment:
There is no any side effect.
Maybe it would be better just:
```
productRouter.get('/products', productController.getAllProducts);
```

Reply:
Done.